### ISSUE TITLE
Doesn't work with icon

### ISSUE THREAD

**Issue by xuwupeng2000** [manzinello/mailgo#23]
so I have an email icon and mailgo doesn't work on it.
```
<a href="mailto:xuwupeng2000@gmail.com">
   <i class="fa fa-3 fa-sub fa-envelope space-required"></i>
</a>
```

**Comment by xuwupeng2000** [comment]
![Screen Shot 2019-05-24 at 4 38 40 PM](https://user-images.githubusercontent.com/2241982/58303066-7c4eef80-7e42-11e9-928c-bd90a974b8f0.png)

http://xuwupeng2000.github.io/

**Comment by manzinello** [comment]
You are right, I've made now some tests and I have found the problem, I will fix it in the next release! Thank you!

**Comment by manzinello** [comment]
Hi! I've fixed it in **mailgo v0.4.7**, released just now! I've tested it, please let me know if everything works now!
